Storage/test.cc

```diff
@@ -48,6 +48,8 @@ namespace transactional_storage_test {
 CURRENT_STRUCT(Element) {
   CURRENT_FIELD(x, int32_t);
   CURRENT_CONSTRUCTOR(Element)(int32_t x = 0) : x(x) {}
+  // Dummy key function to support key-only deleter.
+  int32_t key() const { return x; }
```

Comment:
Speaking of must-have key field - generally yes, but this structure is used in Vector, which doesn't need keys. Are we going to deprecate it?
In general, we'll need composite keys for matrices. So, let's return to macro design question when we implement `ManyToMany` etc.